components/button/button-add.js

```diff
@@ -103,12 +103,7 @@ class ButtonAdd extends RtlMixin(PropertyRequiredMixin(FocusMixin(LocalizeCoreEl
 				0% { position: absolute; }
 				100% { position: static; }
 			}
-
-			button:${unsafeCSS(getFocusPseudoClass())} d2l-button-add-icon-text {
-				background-color: white;
-				border-radius: 0.3rem;
-				box-shadow: 0 0 0 2px var(--d2l-button-add-hover-focus-color);
-			}
+			${getFocusRingStyles(pseudoClass => `button:${pseudoClass} d2l-button-add-icon-text`, { extraStyles: css`background-color: white; border-radius: 0.3rem;` })}
```

Comment:
This is going to be in conflict with @margaree's [change from yesterday](https://github.com/BrightspaceUI/core/pull/5816/).

Reply:
Ah ok I see now! 👍